examples/storage/volume.tf

```diff
@@ -0,0 +1,16 @@
+provider "redfish" {
+  redfish_endpoint = "https://localhost:5000"
+  user = "root"
+  password = "calvin"
+  ssl_insecure = true
+}
+
+resource "redfish_storage_volume" "volume" {
+    storage_controller = "RAID.Integrated.1-1"
+    volume_name = "MyVol"
+    raid_level = "Mirrored"
+    volume_disks = ["Physical Disk 0:1:0", "Physical Disk 0:1:1"]
```

Comment:
These don't look like valid drive IDs, at least for iDRAC. Usually, these are of form ```Disk.Direct.0-0:AHCI.Embedded.1-1``` or ```Disk.Bay.0:Enclosure.Internal.0-1:RAID.Slot.1-1```.